.circleci/config.yml

```diff
@@ -892,16 +892,6 @@ jobs:
           aztec_manifest_key: end-to-end
     <<: *defaults_e2e_test
 
-  integration-archiver-l1-to-l2:
```

Comment:
Nuked the test because archiver became significantly simplified and now it's tested well enough with the other tests.